Marketplace/MarketplaceToolkit.ps1

```diff
@@ -0,0 +1,2109 @@
+                                        </StackPanel>
+
+                                        <StackPanel Orientation="Horizontal" Width="258" Height="25" >
+                                            <Label Content="*" Foreground="red"/>
+                                            <Label Content="Confirm password" />
+                                        </StackPanel>
+                                        <StackPanel Orientation="Horizontal" Width="258" Height="23" Margin="0,0,0,8" ToolTipService.InitialShowDelay="0" ToolTipService.ShowDuration="2147483647" >
+                                            <PasswordBox x:Name="Publish_Pwb_Password2" Width="258" />
+                                            <Button Width="15" Content="!" Foreground="White" Background="Red" BorderThickness="1" BorderBrush="Red" Visibility="Collapsed" Cursor="Hand" />
+                                        </StackPanel>
+
+                                        <StackPanel Orientation="Horizontal" Width="258" Height="25" >
+                                            <Label Content="*" Foreground="red"/>
+                                            <Label Content="API endpoint" />
+                                        </StackPanel>
+                                        <StackPanel Orientation="Horizontal" Width="258" Height="23" Margin="0,0,0,8" ToolTipService.InitialShowDelay="0" ToolTipService.ShowDuration="2147483647" >
+                                            <TextBox x:Name="Publish_Tbx_Endpoint" Width="258" />
+                                            <Button Width="15" Content="!" Foreground="White" Background="Red" BorderThickness="1" BorderBrush="Red" Visibility="Collapsed" Cursor="Hand" />
+                                        </StackPanel>
+
+                                        <Label Content="Publishing log" Margin="0,10,0,10" />
+                                        <ListView x:Name="Publish_Lsv_Log" MinHeight="23" Background="#F9F9F9F9"/>
+                                        <Border BorderBrush="#CCCCCC" BorderThickness="0,1,0,0" Margin="0,15,0,0">
+                                            <Button x:Name="Publish_Btn_Publish" Width="60" Height="23" Content="Publish" HorizontalAlignment="Right" Margin="0,15,0,0"></Button>
+                                        </Border>
+                                    </StackPanel>
+                                </ScrollViewer>
+                            </Grid>
+                        </Border>
+                        <!--#endregion Blade_Upload -->
+                    </DockPanel>
+                </ScrollViewer>
+            </Grid>
+        </DockPanel>
+    </Grid>
+</Window>
+'@
+#endregion
+
+#region Get XAML and create variables
+Add-Type -AssemblyName PresentationFramework
+
+[XML]$XML = $XAML -replace "x:N",'N' 
```

Comment:
If the colon is present the SelectNodes method in line 640 returns no results. Adding the colon to the SelectNodes string return a failure.